build.sbt

```diff
@@ -140,6 +140,14 @@ lazy val actor = pekkoModule("actor")
 lazy val actorTests = pekkoModule("actor-tests")
   .dependsOn(testkit % "compile->compile;test->test", actor)
   .settings(Dependencies.actorTests)
+  .settings(Test / unmanagedSourceDirectories ++= {
```

Comment:
This looks fishy, why are we doing this? Pekko has already been setup to handle all the different types of `scala-*` folders if we wan't to write tests for specific Scala versions and in cases like this I would prefer to KISS

Reply:
the tests in `scala-2.13+` did not execute without this change

Reply:
https://github.com/apache/incubator-pekko/blob/7bee80e05847cd7a1198775b45f831f5ce3227f3/build.sbt#L132-L135 Is probably what you want then

Reply:
That code does not handle the '+' in `2.13+` - I don't want to duplicate the test that is in 2.13+. If I duplicated the code in `scala-3` and `scala-2.13` - sbt handles those source dirs automatically. It doesn't appear to handle `scala-2.13+` automatically.

Reply:
> It doesn't appear to handle `scala-2.13+` automatically.

Indeed sbt doesn't handle this automatically, it would be ideal find the existing code in the build definition that does this so we can reuse it

Reply:
Ah I found it https://github.com/apache/incubator-pekko/blob/main/project/PekkoBuild.scala#L161-L168 . I think if you modify this so it also affects `Test / unmanagedSourceDirectories` then it should fix the same issue in a more principled way.

Reply:
I use IDEA and it is ignoring that file. If I search for strings, it ignores that file - won't even let me open the file to view it. I modified it with a text editor and `sbt ++2.13 "show actor-tests/Test/unmanagedSourceDirectories"` has the right results now. I have committed a change.